ambry-router/src/test/java/com.github.ambry.router/DeleteManagerTest.java

```diff
@@ -62,6 +63,22 @@
   private PartitionId partition;
   private Future<Void> future;
 
+  /**
+   * An checker that either asserts that a delete operation succeeds or returns the specified error code.
+   */
+  private final ErrorCodeChecker defaultErrorCodeChecker = new ErrorCodeChecker() {
```

Comment:
will change, along with the `defaultErrorCodeChecker` -> `getErrorCodeChecker` in the get tests.